src/Components/sharedComponents/ImageBuilderHeader.tsx

```diff
@@ -112,6 +141,34 @@ export const ImageBuilderHeader = ({
                 >
                   Build images
```

Comment:
According to the new mocks this should be moved to the images table's toolbar, though not in this PR scope